RNWCPP/docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to Microsoft/react-native-windows
+
+If this is your first time, below are some more detailed steps for getting your branch set up and submitting a PR. You can either do this with the command prompt or with a combination of the command prompt and SourceTree.
+
+### Setting up your branch for changes 
+ 
+For each bug or task you complete, it is recommended that you start with a fresh branch. If you have any lingering changes in your current branch that you want to save, go ahead and commit them. If you are just beginning, then you are good to go. On github, navigate to your repository which should be forked from Microsoft/react-native-windows. If you haven't already created a fork, follow the steps in **[Git branch Setup](BRANCH_SETUP.md)**. Above the list of files is a dropdown that should say master. Use the dropdown to create a new branch and name is according to what you will be working on. (I.e. DropdownHighlight, CleanUpExamples, etc). Now you have created a new branch. 
+
+**SourceTree:**
+If you are using SourceTree you will want your branch to show up in SourceTree so you can commit changes to your branch. It takes time for it to show up automatically, so you can make it show by running `git remote update origin –prune` in your command prompt from the root. Once you see your new branch in SourceTree under Remotes on the left navigation pane, double click on your branch to check it out locally. A dialog will come up and the default settings should be fine, click Ok.  
+
+**Git Command Line**
+If you are using the command line, you will want to make sure you have your branch locally. It takes time for it to show up automatically, so you can make it show by running `git remote update origin –prune` in your command prompt from the root. Run `git branch -a` to see if your new branch shows up. Now you will want to check out your branch locally. You can do this with `git checkout -b branch-name`. Confirm you are now working out of the branch with `git branch`.
+
+ 
+### Make the fix
+Now that your branch is set up and ready for commits, go ahead and fix the bug you are working on or make some small change that you want to check in. 
+ 
+### Verify your changes
+Manually test your fix, and ensure that all tests are in a good state
```

Comment:
Should there be a detail here on which tests to run? Perhaps @marlenecota can add some preliminary verbiage on the basic tests we want contributors to run until we have a fuller test infrastructure suite set up?